tests/test_tasks.py

```diff
@@ -12,6 +16,35 @@
 has_celery = celery is not None
 
 
+@pytest.mark.django_db
+def test_cleanup_request_logs_command(settings):
+    settings.LOG_OUTGOING_REQUESTS_MAX_AGE = 1  # delete if > 1 day old
+
+    with freeze_time("2023-10-02T12:00:00Z") as frozen_time:
+        OutgoingRequestsLog.objects.create(timestamp=timezone.now())
+        frozen_time.move_to("2023-10-04T12:00:00Z")
+        recent_log = OutgoingRequestsLog.objects.create(timestamp=timezone.now())
+        call_command("cleanup_requests_logs")
```

Comment:
```suggestion
        call_command("cleanup_requests_logs", stdout=String(), stderr=String())
```
typically you want to provide these for `call_command` so that stdout/stderr are not cluttered when running tests

Reply:
Do you mean `StringIO()` or is this `String` something I'm not aware of?

Reply:
StringIO indeed, sorry!